components/colorarea/index.css

```diff
@@ -1,5 +1,6 @@
 %spectrum-ColorControl-hiddenField {
-  opacity: 0.0001;
+  opacity: 0;
```

Comment:
I'm not sure why we had `0.0001`. It shouldn't be needed for accessibility, but it's possible that was the intention. Let me git blame and see who to check with on this.